vm/http.go

```diff
@@ -127,6 +135,26 @@ func builtinHTTPClassMethods() []*BuiltInMethodObject {
 					return t.vm.initStringObject(string(content))
 				}
 			},
+		}, {
+			// Sends a HEAD request to the target with type header and body. Returns the HTTP response as a string.
+			Name: "head",
+			Fn: func(receiver Object) builtinMethodBody {
+				return func(t *thread, args []Object, blockFrame *callFrame) Object {
+					if len(args) != 1 {
+						return t.vm.initErrorObject(errors.ArgumentError, "Expect 1 arguments. got=%v", strconv.Itoa(len(args)))
+					}
+
+					host := args[0].(*StringObject).value
```

Comment:
I think you might need to check the argument's type. Otherwise the Goby will crash if someone accidentally passed an integer variable or something like that.